apps/meteor/tests/end-to-end/api/livechat/12-priorites.ts

```diff
@@ -32,89 +104,102 @@ import { IS_EE } from '../../../e2e/config/constants';
 				.get(api('livechat/priorities'))
 				.set(credentials)
 				.expect('Content-Type', 'application/json')
 				.expect(200);
 			expect(response.body).to.have.property('success', true);
-			expect(response.body.priorities).to.be.an('array').with.lengthOf.greaterThan(0);
-			expect(response.body.priorities.find((p: IOmnichannelServiceLevelAgreements) => p._id === priority._id)).to.be.an('object');
+			expect(response.body.priorities).to.be.an('array');
+			expect(response.body.priorities).to.have.length.greaterThan(0);
+			expect(response.body.priorities.find((p: ILivechatPriority) => !!p && p?._id === priority._id)).to.have.property('_id', priority._id);
+		});
+		it('should not create the same priority twice', async () => {
+			await updatePermission('manage-livechat-priorities', ['admin', 'livechat-manager']);
+			await updatePermission('view-l-room', ['livechat-agent']);
+			const newPriority = await savePriority({ name: priority.name, level: priority.level });
+			expect(newPriority).not.to.have.property('_id', priority._id);
+			expect(newPriority).not.to.have.property('name', priority.name);
+			expect(newPriority).not.to.have.property('level', priority.level);
+			expect(newPriority).to.have.property('success', false);
+			expect(newPriority).to.have.property('error');
 		});
 	});
 
-	describe('livechat/priorities/:priorityId', () => {
+	describe('livechat/priority/:priorityId', () => {
 		it('should return an "unauthorized error" when the user does not have the necessary permission', async () => {
 			await updatePermission('manage-livechat-priorities', []);
 			await updatePermission('view-l-room', []);
-			const response = await request
-				.get(api('livechat/priorities/123'))
-				.set(credentials)
-				.expect('Content-Type', 'application/json')
-				.expect(403);
+			const response = await request.get(api('livechat/priority/123')).set(credentials).expect(403);
 			expect(response.body).to.have.property('success', false);
+			expect(response.body).to.have.property('error');
+			expect((response.body as any)?.error).to.contain('error-unauthorized');
 		});
-		it('should return a priority', async () => {
-			await updatePermission('manage-livechat-priorities', ['admin']);
+		it('should create, find and delete a priority', async () => {
+			await updatePermission('manage-livechat-priorities', ['admin', 'livechat-manager']);
 			await updatePermission('view-l-room', ['livechat-agent']);
 			const priority = await savePriority();
 			const response = await request
-				.get(api(`livechat/priorities/${priority._id}`))
+				.get(api(`livechat/priority/${priority._id}`))
 				.set(credentials)
 				.expect('Content-Type', 'application/json')
 				.expect(200);
 			expect(response.body).to.have.property('success', true);
 			expect(response.body).to.be.an('object');
 			expect(response.body._id).to.be.equal(priority._id);
+			const deletable = deletePriority(priority._id);
+			expect(deletable).to.not.rejected;
```

Comment:
This test can be separated in 2. One for create/fetch and other for create/delete